.circleci/config.yml

```diff
@@ -635,6 +674,249 @@ jobs:
           command: |
             echo Hello Circle!
 
+  ask-for-approvals:
+    machine:
+      image: default # latest stable version
+    steps:
+    - attach_workspace:
+        at: /tmp/workspace
+    - run:
+        name: Load JIRA issue key
+        command: |
+          if [ -f /tmp/workspace/jira-env.sh ]; then
+            source /tmp/workspace/jira-env.sh
+            echo "Loaded JIRA_ISSUE_KEY: $JIRA_ISSUE_KEY"
+          else
+            echo "ERROR: jira-env.sh not found"
+            echo "This file is required for this deployment. It contains the Jira issue key associated with this release"
+            exit 1
+          fi
+    - run:
+        name: Comment on Jira
+        command: |
+          curl --request POST \
+            --url https://precisionmedicineinitiative.atlassian.net/rest/api/3/issue/${JIRA_ISSUE_KEY}/comment \
+            --header 'accept: application/json' \
+            --header "authorization: Basic ${JIRA_KEY}" \
+            --header 'content-type: application/json' \
+            --data '{
+            "body": {
+              "content": [
+                {
+                  "content": [
+                    {
+                      "text": "This is a test. ",
+                      "type": "text"
+                    }
+                  ],
+                  "type": "paragraph"
+                },
+                {
+                  "content": [
+                    {
+                      "text": "Requesting change approval for this release of the Workbench to production. 1 security and 1 business approval are needed for release",
+                      "type": "text"
+                    }
+                  ],
+                  "type": "paragraph"
+                },
+                {
+                  "content": [
+                    {
+                      "type": "mention",
+                      "attrs": {
+                        "id": "5b50af8f10d57114135ef067",
+                        "text": "Charissa Rotundo"
+                      }
+                    },
+                    {
+                      "text": " or ",
+                      "type": "text"
+                    },
+                    {
+                      "type": "mention",
+                      "attrs": {
+                        "id": "5ea1be1e61f0e70c11448011",
+                        "text": "Marcie Ansell"
+                      }
+                    },
+                    {
+                      "text": " (security)",
+                      "type": "text"
+                    }
+                  ],
+                  "type": "paragraph"
+                },
+                {
+                  "content": [
+                    {
+                      "type": "mention",
+                      "attrs": {
+                        "id": "557058:706ddbed-1979-4f5b-845e-29c4a92b3074",
```

Comment:
how did you find these IDs?

This one has a different format.  do you know why?

Reply:
If you go to a user's profile page, it is in the URL.

Asmita's looks different. Maybe it's newer or older? Not sure.

![Screenshot 2025-04-09 at 1 00 36 PM](https://github.com/user-attachments/assets/46005235-7ab5-417e-a8fd-2c414f8358ad)